RELEASE_NOTES.md

```diff
@@ -40,6 +40,7 @@
 - The `podman stop` command now includes a new flag, `--filter`, to filter which containers will be stopped (e.g. `podman stop --all --filter label=COM.MY.APP`).
 - The Podman global option `--url` now has two aliases: `-H` and `--host`.
 - The `podman network create` command now supports a new option with the default `bridge` driver, `--opt isolate=`, which isolates the network by blocking any traffic from it to any other network with the `isolate` option enabled. This option is enabled by default for networks created using the Docker-compatible API.
+- The `podman image sign` command now supports creating a signature file that can be used later to verify the image.
```

Comment:
I don't see changes to `image sign`.  Is it meant to describe the sigstore changes?  Those went into `push`:

```suggestion
- Added the ability to create sigstore signatures in `podman push` and `podman manifest push`.
- Added an option to read image signing passphrase from a file.
```